stix_shifter/stix_transmission/src/modules/guardium/guardium_error_mapper.py

```diff
@@ -0,0 +1,44 @@
+from .....utils.error_mapper_base import ErrorMapperBase
+from .....utils.error_response import ErrorCode
+
+error_mapping = {
+        #search does not exist
+        1002: ErrorCode.TRANSMISSION_SEARCH_DOES_NOT_EXISTS,
+        # The search cannot be created. The requested search ID that was provided in the query expression is already in use.
+        # Please use a unique search ID (or allow one to be generated).
+        1004: ErrorCode.TRANSMISSION_MODULE_DEFAULT_ERROR.value,
+        # A request parameter is not valid
+        1005: ErrorCode.TRANSMISSION_INVALID_PARAMETER,
+        #The Ariel server might be temporarily unavailable or offline. Please try again later.
+        1010: ErrorCode.TRANSMISSION_REMOTE_SYSTEM_IS_UNAVAILABLE,
+        #An error occurred during the attempt
+        1020: ErrorCode.TRANSMISSION_MODULE_DEFAULT_ERROR.value,
+        #error in AQL
+        2000: ErrorCode.TRANSMISSION_QUERY_PARSING_ERROR,
+        #error in AUTHORIZATION CREDENTIAL
+        3001: ErrorCode.TRANSMISSION_AUTH_CREDENTIALS,
+        #error in ACCESS TOKEN MISSING
+        3002: ErrorCode.TRANSMISSION_AUTH_CREDENTIALS
+    }
```

Comment:
this error_mapping dictionary most likely copied from QRadar module. please update the comments. i see ariel/aql mentioned in few places.

Reply:
updated and modified for guardium